apps/fluent-tester/src/RNTester/TestComponents/Test.tsx

```diff
@@ -68,9 +69,23 @@ const styles = StyleSheet.create({
 });
 
 export const Test = (props: TestProps) => {
+
+  const themeColor = () => {
```

Comment:
So as part of the TODO/next steps we had mentioned filing as in issue I would two things related to this
1. We should convert the tester to use FluentUI Text and work through the bugs we had been seeing
2. Those Text components and anything else shouldn't need to have a theme color layered on style, in theory they'd respond to the theme changes as long as the entire tester is wrapped in a themeprovider